specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json

```diff
@@ -80,53 +80,12 @@
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/contentTypes/{contentTypeId}": {
-      "head": {
```

Comment:
>head [](start = 7, length = 4)

This looks like a potentially breaking change as per [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Do you have a breaking change approval for it or can you explain why it's not a breaking change?

Reply:
This is a new api-version and not published yet.

Reply:
Read the link I sent. Yes, breaking changes require an new API version, which you are doing but removing things across API versions is also considered a breaking change and requires separate approval (instructions in the link).

---
In reply to: [466675140](https://github.com/Azure/azure-rest-api-specs/pull/10328#discussion_r466675140) [](ancestors = 466675140)